FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,48 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			guard self != nil else {return}
+			switch result {
+			case .success((let data, let response)):
```

Comment:
The `let` can be defined outside the tuple so the expression can be shorter:
```suggestion
			case let .success((data, response)):
```